starter-code/index.js

```diff
@@ -1,10 +1,46 @@
-var SortedList = function() {};
-
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+var SortedList = function() {
+    this.items = [];
+    this.length = this.items.length;
+};
+
+SortedList.prototype.add = function(item) {
+    this.items.push(item);
+    this.length += item;
+};
+SortedList.prototype.get = function(pos) {
+    this.items.length
+    return this.items[pos-1];
```

Comment:
this is ok! :+1: (I WAS WRONG! when reviewing in class) after checking a bit more the tests, just realised that it compares the values and expects them to be equal.